Panic when creating sdk hosted terminator

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb8 pc=0x1bc083e]
goroutine 325 [running]:
github.com/openziti/fabric/controller/command.(*LocalDispatcher).Dispatch.func1()
github.com/openziti/fabric@v0.20.0/controller/command/command.go:61 +0x2d4
panic({0x217e540, 0x367a440})
runtime/panic.go:884 +0x212
github.com/openziti/edge/controller/model.(*CreateEdgeTerminatorCmd).getTerminatorSession(0xc000bf9fc8, 0x7fc66021fe30?, {0x28b3758, 0xc0028c42d0}, {0x243d3da, 0x8})
github.com/openziti/edge@v0.23.0/controller/model/create_terminator_cmd.go:84 +0x33e
github.com/openziti/edge/controller/model.(*CreateEdgeTerminatorCmd).validateTerminatorIdentity(0xc000bf9fc8, 0x28b6c20?, 0xc002ea9a40)
github.com/openziti/edge@v0.23.0/controller/model/create_terminator_cmd.go:42 +0x138
github.com/openziti/fabric/controller/network.(*TerminatorManager).ApplyCreate.func1(0xc003348700)
github.com/openziti/fabric@v0.20.0/controller/network/terminator.go:146 +0x24f
go.etcd.io/bbolt.(*DB).Update(0x18?, 0xc000bf9fe0)
go.etcd.io/bbolt@v1.3.6/db.go:741 +0x82
github.com/openziti/storage/boltz.(*DbImpl).Update(0x2226860?, 0xc003347601?)
github.com/openziti/storage@v0.1.21/boltz/db.go:103 +0x96
github.com/openziti/fabric/controller/network.(*TerminatorManager).ApplyCreate(0xc000212f40, 0xc000bb2de0)
github.com/openziti/fabric@v0.20.0/controller/network/terminator.go:138 +0x8b
github.com/openziti/edge/controller/model.(*CreateEdgeTerminatorCmd).Apply(0xc000bf9fc8)
github.com/openziti/edge@v0.23.0/controller/model/create_terminator_cmd.go:27 +0x105
github.com/openziti/fabric/controller/command.(*LocalDispatcher).Dispatch(0xc00031d110?, {0x28aa650?, 0xc000bf9fc8?})
github.com/openziti/fabric@v0.20.0/controller/command/command.go:77 +0x147
github.com/openziti/fabric/controller/network.(*Managers).Dispatch(...)
github.com/openziti/fabric@v0.20.0/controller/network/managers.go:58
github.com/openziti/edge/controller/handler_edge_ctrl.(*createTerminatorHandler).CreateTerminator(0xc00110ecc0, 0xc00229e5a0)
github.com/openziti/edge@v0.23.0/controller/handler_edge_ctrl/create_terminator.go:116 +0x9a2
created by github.com/openziti/edge/controller/handler_edge_ctrl.(*createTerminatorHandler).HandleReceive
github.com/openziti/edge@v0.23.0/controller/handler_edge_ctrl/create_terminator.go:68 +0x2df

--- a/controller/persistence/session_store.go
+++ b/controller/persistence/session_store.go
@@ -17,10 +17,10 @@
 package persistence
 
 import (
-	"github.com/openziti/storage/ast"
-	"github.com/openziti/storage/boltz"
 	"github.com/openziti/foundation/v2/errorz"
 	"github.com/openziti/foundation/v2/stringz"
+	"github.com/openziti/storage/ast"
+	"github.com/openziti/storage/boltz"
 	"github.com/pkg/errors"
 	"go.etcd.io/bbolt"
 	"time"
@@ -198,7 +198,7 @@ func (store *sessionStoreImpl) LoadOneByToken(tx *bbolt.Tx, token string) (*Sess
 	if id != nil {
 		return store.LoadOneById(tx, string(id))
 	}
-	return nil, nil
+	return nil, boltz.NewNotFoundError(store.GetSingularEntityType(), "token", token)
 }
 
 func (store *sessionStoreImpl) LoadCerts(tx *bbolt.Tx, id string) ([]*SessionCert, error) {

